### ISSUE TITLE
Bug in demo

### ISSUE THREAD

**Issue by isaacperez** [mlzxy/devit#3]
I have downloaded everything but when I try the demo I get the same bounding box but the wrong labels:
![ycb out](https://github.com/mlzxy/devit/assets/12383906/52b01c89-0b09-4864-b528-e4ae1ab1863d)

**Comment by isaacperez** [comment]
I set `category_space=None` and that was the problem, doesn't the trained model work without creating the file for `category_space`?

**Comment by zhszysrh** [comment]
i download the files below and run the demo command without anything change，i get the output like the second picture，it seems normal
![image](https://github.com/mlzxy/devit/assets/107545756/0f886aaa-91a6-45ae-9362-b62f9794a111)
![image](https://github.com/mlzxy/devit/assets/107545756/da922a14-1782-4333-8398-a2cc18bd86b6)

**Comment by isaacperez** [comment]
That means you have used `category_space="demo/ycb_prototypes.pth"`. So you have overwritten the model classes. If you leave `category_space=None`, the labels look random, shouldn't they be COCO or LVIS classes?

**Comment by mlzxy** [comment]
> That means you have used `category_space="demo/ycb_prototypes.pth"`. So you have overwritten the model classes. If you leave `category_space=None`, the labels look random, shouldn't they be COCO or LVIS classes?

Hi @isaacperez , the labels seem to be from LVIS if you set category space as none. LVIS has lots of long-tailed classes.

**Comment by isaacperez** [comment]
Hi, in that case the model file does not contain the correct prototypes/classes (see the image at the beginning of the issue) because, for example, it is labeling a pringles can as a control.

**Comment by isaacperez** [comment]
Do you think that is possible with LVIS prototypes?

**Comment by mlzxy** [comment]
> Do you think that is possible with LVIS prototypes?

Yes. Actually this demo is a highly challenging scenario, more difficult than COCO or LVIS I would say because of occlusion, very crowded scene, retailed categories (retail objects recognition is in itself hard) and etc. 

So ideally when using LVIS prototypes, it shall not produce as many boxes, I haven't tried this either but turns out it does. This is our limitation for sure. This could very likely motivates future work on how to preventing false positive in open-set detection, how to handle retail categories and etc.
